TIME_OF_FREE_FALL.s

```diff
@@ -139,7 +139,7 @@ TFFTEM		=	36D	#	TEMPORARY
 #		TFFNP		E:(-38+2NR)		M  LCP, SEMI LATUS RECTUM, WEIGHTED BY NR.  FOR VGAMCALC.
 #				M:(-36+2NR)
 #		TFF/RTMU	E:(17) M:(14)		1/SQRT(MU)
-#		TFFVSQ		E:(20) M:(18)		1/M  -(V SQ/MU):  PRESENT VELOCITY, NORMLIZED.  FOR VGAMCALC
+#		TFFVSQ		E:(20) M:(18)		1/M  -(V SQ/MU):  PRESENT VELOCITY, NORMALIZED.  FOR VGAMCALC
```

Comment:
`NORMLIZED` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1375.jpg). There should also not be a space before this word - i.e `VELOCITY,NORMLIZED`